docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -94,31 +92,12 @@ Key:       /Users/alice/.tsh/keys/teleport.example.com/alice
 The displayed `CA`, `Cert`, and `Key` files are used to connect through pgAdmin
 4, MySQL Workbench, and other graphical database clients that support mutual
 TLS authentication.
-</TabItem>
-</Tabs>
-
-</TabItem>
-<TabItem scope="cloud" label="Teleport Enterprise Cloud">
```

Comment:
The Cloud tab was missing "Authenticated Proxy" option. Instead of adding the option here, i have dropped the nested tab and added a note in the "separate ports" option that it won't work for cloud. 

Open to suggestions